### PR TITLE
Alter query initialization to show comments

### PR DESCRIPTION
**Pull request by kierabr1**
I got rid of timestamp in the query initialization which fixed my error and allowed all previously entered comments to show.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/Entry.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** An item on a todo list. */
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/data/Entry.java:17]: Update the comment to say something like

'Class for specifying comment text and metadata'or something like that

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -30,25 +33,34 @@
...
+    PreparedQuery results = datastore.prepare(query);
+    List<Entry> entries = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+      String name = (String) entity.getProperty("name");
+      String email = (String) entity.getProperty("email");
+      String comment = (String) entity.getProperty("comment");
+      Entry entry = new Entry(name, email, comment);
+      entries.add(entry);
+    }
+    System.out.println(entries);
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:49]: You should move everything from Query query till here to a separate method,

private List<Entry> entries = readFromDatastore();

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -27,23 +30,19 @@ function addRandomFunFact() {
   funfactContainer.innerText = funfact;
 }
 
-// TODO: Use following functions in step 5 to display comment
-
+// TODO: Use following functions in step 5 to display comment.
```
Comment (sbahaud) [inline portfolio/src/main/webapp/script.js:33]: Remove this comment now that you are displaying comments

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -27,23 +30,19 @@ function addRandomFunFact() {
...
+    console.log(entries);
+    const quoteListElement = document.getElementById('entry-container');
+    entries.forEach((entry) => {
+      quoteListElement.appendChild(createQuoteElement('Name: ' + entry.name + ': ' + entry.comment));
+    })
   });
 }
 
- function createQuoteElement(text) {
+function createQuoteElement(text) {
```
Comment (sbahaud) [inline portfolio/src/main/webapp/script.js:44]: nit: I would call this function just createListElement, list there nothing Quote specific in the function.

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -27,23 +30,19 @@ function addRandomFunFact() {
...
-  const quoteListElement = document.getElementById('entry-container');
-  quoteListElement.appendChild(
-      createQuoteElement('Name: ' + entries.name));
-  quoteListElement.appendChild(
-      createQuoteElement('Email: ' + entries.email));
-  quoteListElement.appendChild(
-      createQuoteElement('Comment: ' + entries.comment));
+    console.log(entries);
+    const quoteListElement = document.getElementById('entry-container');
+    entries.forEach((entry) => {
```
Comment (sbahaud) [inline portfolio/src/main/webapp/script.js:38]: rename quoteListElement to commentListElement.

**Review by sbahaud — APPROVED** [review]
LGTM! Approving with some a few minor comments.

File: portfolio/src/main/java/com/google/sps/data/Entry.java (hunk 1 above)
Comment (kierabr1) [reply portfolio/src/main/java/com/google/sps/data/Entry.java:17]: Done

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (kierabr1) [reply portfolio/src/main/webapp/script.js:44]: Done

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (kierabr1) [reply portfolio/src/main/webapp/script.js:33]: Done

File: portfolio/src/main/webapp/script.js (hunk 5 above)
Comment (kierabr1) [reply portfolio/src/main/webapp/script.js:38]: Done

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (kierabr1) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:49]: Done

**Pull request merged**